coaster/sqlalchemy/statemanager.py

```diff
@@ -151,6 +159,12 @@ def redraft(self):
     post.publish()          # Change state from DRAFT to PUBLISHED
     post.state.RECENT       # True (this one calls the validator if the base state matches)
 
+Transitions return a tuple to show if they succeeded. If not, they'll return the reason.
```

Comment:
Tuples are a project convention, not an expectation from StateManager. This should not be in the documentation.